modules/op-release-notes-1-7.adoc

```diff
@@ -0,0 +1,307 @@
+
+* The `Conditions` custom resource definition (CRD) type will be removed in a future release. Instead, use the recommended `When` expression.
+// issue # unknown; discussed in Slack.
+
+// Triggers
+* Breaking change: Triggers validates and gives error if you provide empty `EventListener` and `TriggerBinding` specifications.
+// https://github.com/tektoncd/triggers/pull/1277 and https://github.com/tektoncd/triggers/pull/1264
+
+
+[id="known-issues-1-7_{context}"]
+== Known issues
+
+// Pipelines
+* Implicit parameter mapping incorrectly passes parameters from the top-level `Pipeline` or `PipelineRun` definitions to the `taskRef` tasks. Mapping should only occur from a top-level resource to tasks with in-line `taskSpec` specifications. This issue only affects users who have set `enable-api-fields` feature flag to `alpha`.
+// https://github.com/tektoncd/pipeline/issues/4483
+
+
+
+[id="fixed-issues-1-7_{context}"]
+== Fixed issues
+
+// Pipelines
+* With this fix, metadata such as `labels` and `annotations` from task and pipeline definitions does not propagate to task runs and pipeline runs.
+// https://github.com/tektoncd/pipeline/pull/4478
+
+* With this fix, if the `timeouts.tasks` field or the `timeouts.finally` field is set to `0`, then the `timeouts.pipeline` is also set to `0`. 
+// https://github.com/tektoncd/pipeline/pull/4539
+
+* With this fix, the `-x` set flag is removed from scripts that do not use a shebang. The fix reduces potential data leak from script execution.
+// https://github.com/tektoncd/pipeline/pull/4451
+
+* With this fix, any backslash character present in the usernames in Git credentials will be escaped with an additional backslash in the `.gitconfig` file.
+// https://github.com/tektoncd/pipeline/pull/4337
+
+// Triggers
+* With this fix, the `finalizer` property of the `EventListener` object is not necessary for cleaning up logging and config maps.
+// https://github.com/tektoncd/triggers/pull/1244
+
+* With this fix, the default HTTP client associated with the event listener server is removed, and a custom HTTP client added. As a result, the timeouts have improved.
+// https://github.com/tektoncd/triggers/pull/1308
+
+* With this fix, the Triggers cluster role now works with owner references.
+// https://github.com/tektoncd/triggers/pull/1267
+
+* With this fix, the race condition in the event listener does not happen when multiple interceptors return extensions.
+// https://github.com/tektoncd/triggers/pull/1282
+
+// CLI
+* With this fix, the `tkn pr delete` command does not the pipeline runs with the `ignore-running` flag.
+// https://github.com/tektoncd/cli/pull/1532
+
+// Operator
+* With this fix, the Operator pods will not continue restarting when you modify any add-on parameters.
+// https://github.com/tektoncd/operator/pull/631
+
+* With this fix, the `tkn serve` CLI pod will be scheduled on infra nodes, if not configured in the subscription and config custom resources.
```

Comment:
```suggestion
* With this fix, the `tkn serve` CLI pod is scheduled on infra nodes, if not configured in the subscription and config custom resources.
```
Avoid "will"/future tense.

Reply:
```suggestion
* With this update, if you do not configure the `tkn serve` CLI pod in the subscription and config custom resources, the scheduler runs it on the infra nodes.
```
- Please use the standard release note templates. If not, adopt some language from them. 
- Also, it's hard for me to understand this release note. This suggestion is my best guess at the intended meaning.